src/tests/dbus-tests/test_30_iscsi.py

```diff
@@ -374,3 +374,56 @@ def test_login_noauth_badauth(self):
         udisks = self.get_object('')
         objects = udisks.GetManagedObjects(dbus_interface='org.freedesktop.DBus.ObjectManager')
         self.assertNotIn(dbus_path, objects.keys())
+
+    def test_ibft(self):
```

Comment:
I think we should tag this test as `UNSAFE`, just to be sure we won't try to login to some ibft iscsi when running in some random environment.

Reply:
Yeah, good idea. I don't expect these tests to be ever used except of very isolated cases of specific testing environments.